doc/en/components/general-nuget-feed.md

```diff
@@ -38,6 +41,10 @@ This topic contains the following sections:
 3 - Execute the following command
 
 ```cmd
+//nuget protocol v3
+nuget sources add -name "Infragistics" -source "https://packages.infragistics.com/nuget/licensed/v3/index.json" -username "your login email" -password "your password"
+
+//nuget protocol v2 or later
```

Comment:
I would word this as just "Nuget protocol v2".
The "or later" bit incorrectly implies later versions of the NuGet HTTP protocol. I'd remove the "or later" part as it does not add anything in this context.